recipes/pytest-monitor/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pytest-monitor" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CFMTech/pytest-monitor/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: b4081347adcbc13c23eac3384a3dfa83a1c6301bb3aaae90e5588ec30a231ecd
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install --no-deps  . -vv"
+
+requirements:
+  host:
+    - python
```

If you want to limit the Python version, you can use in `host` and `run` the following:
```suggestion
    - python >=3.6
```

---

Noted, thanks!